provisioning/stream/kubernetes/templates/http-source/deployment.yaml

```diff
@@ -0,0 +1,129 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: stream-http-source
+  namespace: $NAMESPACE
+  labels:
+    app: stream-http-source
+    releaseId: $RELEASE_ID
+spec:
+  replicas: $STREAM_HTTP_SOURCE_REPLICAS
+  selector:
+    matchLabels:
+      app: stream-http-source
+  template:
+    metadata:
+      labels:
+        app: stream-http-source
+        releaseId: $RELEASE_ID
+        stream-etcd-client: "true"
+        tags.datadoghq.com/env: "$KEBOOLA_STACK"
+        tags.datadoghq.com/service: "stream"
```

Comment:
dal bych stream-http-source

Reply:
Je to jedna `stream` service, ktora ma viacero podov:
- V DD by sme to potrebovali mat spolu.
- Da sa to rozdelit na este viac podov/casti - pre lepsie skalovanie, alebo sa niektore mozno zlucia pre efektivnejsie vyuzivanie resources, a nechceme stale vyvarat v DD nove services.